osu.Server.Spectator/Database/Models/matchmaking_room_event.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+
+namespace osu.Server.Spectator.Database.Models
+{
+    // ReSharper disable InconsistentNaming
+
+    public class matchmaking_room_event
+    {
+        public long id { get; set; }
+        public long room_id { get; set; }
+        public string event_type { get; set; } = string.Empty;
+        public long? playlist_item_id { get; set; }
```

Comment:
Not 100% this is going to be convenient, because to get to any stats like pick% you'll have to do a join over this every time to `multiplayer_playlist_items`, which may end up being slow, but if it works for you...